### ISSUE TITLE
Running it on colab, settings don't get saved

### ISSUE THREAD

**Issue by ytt246** [henk717/KoboldAI-Client#317]
For example, when I'm running the official version of KoboldAI, my active soft prompt will be saved and will be activated automatically the next time I start it. But when I'm running this the soft prompt will always be none by default and have to be activated manually

**Comment by henk717** [comment]
Are you running our official notebook with google drive enabled?

**Comment by ytt246** [comment]
Yes, Pygmalion 6B and United. When I go to google drive I can see the settings files are getting updated, but next time I load up soft prompt is still not selected

**Comment by ebolam** [comment]
Is it just the soft prompt not being loaded? I'm not sure that setting is saved
